app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -345,24 +345,6 @@ export default class SampleViewV2 extends React.Component {
   getTaxonMetricValue = (row, metric) => {
     let parsedMetric = metric.split(":");
     let parsedValue = get(parsedMetric, row);
-
-    // Contigs/contig_reads are stored in the format {length of contig: count},
-    // so some extra processing is needed to extract the value.
-    if (parsedMetric.includes("contigs") && parsedValue) {
```

Comment:
I'm not sure whether there is more special casing of contigs elsewhere or how to search for it. Please advise @tfrcarvalho .

Reply:
Not sure either. But I cannot find any other reference.